app/themes/user-admin/e2e/login/login.e2e-spec.ts

```diff
@@ -0,0 +1,77 @@
+import { UserAdminPage } from './login.po';
+import { protractor, browser, element, by, WebElement } from 'protractor';
+
+describe('login page', () => {
+  let page: UserAdminPage;
+
+  beforeEach(() => {
+    page = new UserAdminPage();
+  });
+
+  /*
+    Initial home page (login form) elements
+  */
+
+  it('should display message saying User Admin', () => {
+    page.navigateTo();
+    expect(page.getParagraphText(".navbar-brand")).toEqual('User Admin');
+  });
+
+  it('should display a login email and password input element', () => {
+    expect(element(by.css('input[name=email]'))).toBeTruthy();
+    expect(element(by.css('input[name=password]'))).toBeTruthy();
+  });
+
+  it('should display a submit button', () => {
+    expect(element(by.css('btn btn-default'))).toBeTruthy();
+  });
+
+  /*
+    Login form negative case
+  */
+  it('should display a login fail message', () => {
+    let email = browser.findElement(protractor.By.name('email'));
+    let password = browser.findElement(protractor.By.name('password'));
+    let submit = browser.findElement(protractor.By.className('btn btn-success'));
+
+    // Input Values
+    email.sendKeys('foo@bar.com');
+    password.sendKeys('barfood');
```

Comment:
here is another example of inconsistent test username/passwords. see note on `home.e2e-spec.ts` about moving the login to a globally used fixture method.